framework/include/problems/FEProblemBase.h

```diff
@@ -1614,6 +1614,11 @@ class FEProblemBase : public SubProblem, public Restartable
     _error_on_jacobian_nonzero_reallocation = state;
   }
 
+  /**
+   * Whether or not we are allowing invalid solutions
+   */
+  bool allowInvalidSolution() { return _allow_invalid_solution; }
```

Comment:
Can be `const`

Reply:
I'll update this in the next version of the feature.

Reply:
I'm very impressed with our quickly you're contributing @MengnanLi91!

Reply:
It must be the NCSU power 😄

Reply:
Go wolfpack! I also got a lot of help from our awesome team to get my first PR done : )
![image](https://user-images.githubusercontent.com/118846840/207951373-eb93f8ba-f30f-4e9b-a839-50504cff38f6.png)